src/ML-POD/compute_podfit.h

```diff
@@ -0,0 +1,214 @@
+/***************************************************************************
+               CESMIX-MIT Project
+
+ Contributing authors: Ngoc-Cuong Nguyen (cuongng@mit.edu, exapde@gmail.com)
+ ***************************************************************************/
+
+#ifdef COMPUTE_CLASS
+// clang-format off
+ComputeStyle(podfit,CPODFIT);
+// clang-format on
+#else
+
+#ifndef LMP_COMPUTE_PODFIT_H
+#define LMP_COMPUTE_PODFIT_H
+
+#include "compute.h"
+#include "pod.h"
+
+namespace LAMMPS_NS {
+
+class CPODFIT : public Compute {
```

Comment:
The naming convention for compute style classes is ComputeName.

I don't see any "compute" member function declaration here. What is the "consumable property" that this compute style computes, is it a global scalar, a global vector, or a local vector/array or a per-atom vector/array?
How is the compute triggered and how the results accessed? I cannot find anything useful in the documentation either. That one reads like this "compute" should rather be a CommandStyle class. Please explain.

Reply:
We use CommandStyle class for fitpod.